.github/workflows/release.yml

```diff
@@ -185,7 +185,7 @@ jobs:
           JIRA_DESCRIPTION: ${{ env.jira_description }}
```

Comment:
The changes made in this code update the format of the RELEASE_NOTES variable by removing the extra line breaks and changing the formatting. The sed commands used to manipulate the text seem appropriate for the intended changes. No issues found. LGTM